Sources/Tracing/Tracer.swift

```diff
@@ -47,25 +53,111 @@ public protocol Tracer: Instrument {
 }
 
 extension Tracer {
-    /// Start a new ``Span`` with the given `Baggage` starting at `DispatchWallTime.now()`.
+    #if swift(>=5.3.0)
+    /// Start a new ``Span`` with the given `Baggage` starting "now".
     ///
     /// - Parameters:
     ///   - operationName: The name of the operation being traced. This may be a handler function, database call, ...
     ///   - baggage: Baggage potentially containing trace identifiers of a parent ``Span``.
     ///   - kind: The ``SpanKind`` of the ``Span`` to be created. Defaults to ``SpanKind/internal``.
+    ///   - function: The function name in which the span was started
+    ///   - fileID: The `fileID` where the span was started.
+    ///   - line: The file line where the span was started.
     public func startSpan(
         _ operationName: String,
         baggage: Baggage,
-        ofKind kind: SpanKind = .internal
+        ofKind kind: SpanKind = .internal,
+        function: String = #function,
+        file fileID: String = #fileID,
```

Comment:
I'm very much looking forward to a time where we can start to only support `#fileID` in server-side Swift projects 😬

Reply:
Tbh NIO requires 5.5+ already, so most projects can. I'm not even sure we should keep trying with 5.1+ here... but for now we do.